src/js/controllers/ctrl-create.js

```diff
@@ -158,6 +158,15 @@ app.controller('createCtrl', function(
 		}
 	}
 
+	const canPublish = () => {
```

Comment:
name could be a little better - I'd like to know this is touching the api without scrolling to it